### PR TITLE
PSR-2138 fixed prod Issue on shares disposals

### PR DESCRIPTION
**Pull request by narendrapaduchuri**

### CODE REVIEW

File: app/utils/nonsipp/summary/BondsDisposalCheckAnswersUtils.scala [hunk 1]
```diff
@@ -86,8 +86,8 @@ object BondsDisposalCheckAnswersUtils
     request.userAnswers
       .map(BondsDisposalProgress.all())
       .flatMap { case (bIndex, disposals) =>
-        disposals.map { case (dIndex, progress) =>
-          ((bIndex, dIndex), progress)
+        disposals.collect { case (dIndex, SectionJourneyStatus.Completed) =>
```
Comment (kberec) [inline app/utils/nonsipp/summary/BondsDisposalCheckAnswersUtils.scala:89]: Do we need unit tests for these? I.e. to make sure in progress records are not fetched.

Comment (narendrapaduchuri) [reply app/utils/nonsipp/summary/BondsDisposalCheckAnswersUtils.scala:89]: Hi @kberec , sure will check and fix it.

Comment (jamalosman) [reply app/utils/nonsipp/summary/BondsDisposalCheckAnswersUtils.scala:89]: We currently have one test where all journeys are completed. We can create a couple more tests to cover more cases line in progress and none recorded.

**Review by jamalosman — APPROVED** [review]

**Review by MadStu — APPROVED** [review]
